airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/base/ssh/SshBastionContainer.java

```diff
@@ -36,21 +36,27 @@ public void initAndStartBastion(final Network network) {
     bastion.start();
   }
 
+  public JsonNode getTunnelMethod(final SshTunnel.TunnelMethod tunnelMethod,
+                                  final boolean innerAddress)
+      throws IOException, InterruptedException {
+    final var containerAddress = innerAddress ? getInnerContainerAddress(bastion) : getOuterContainerAddress(bastion);
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put("tunnel_host",
+            Objects.requireNonNull(containerAddress.left))
+        .put("tunnel_method", tunnelMethod)
+        .put("tunnel_port", containerAddress.right)
+        .put("tunnel_user", SSH_USER)
+        .put("tunnel_user_password", tunnelMethod.equals(SSH_PASSWORD_AUTH) ? SSH_PASSWORD : "")
+        .put("ssh_key", tunnelMethod.equals(SSH_KEY_AUTH) ? bastion.execInContainer("cat", "var/bastion/id_rsa").getStdout() : "")
+        .build());
+  }
```

Comment:
This got lifted out of `getTunnelConfig` because we can't use the latter with the new `TestDatabase.ConfigBuilder`  objects.